src/ggml.c

```diff
@@ -7166,6 +7170,73 @@ struct ggml_tensor * ggml_conv_transpose_2d_p0(
     return result;
 }
 
+
+// ggml_winograd
+
+// a: [OC，IC, 3, 3]
+// result: [OC, IC, 16]
+struct ggml_tensor * ggml_winograd_stage0(
+        struct ggml_context * ctx,
+        struct ggml_tensor  * a) {
+    bool is_node = false;
+ 
+    if (a->grad) {
+        is_node = true;
+    }
```

Comment:
If https://github.com/ggerganov/ggml/pull/966 is merged first this will need to be removed (should be very straightforward).

Reply:
Look forward to it...